idl/internal/docstring.go

```diff
@@ -0,0 +1,138 @@
+package internal
+
+import (
+	"strings"
+	"unicode"
+)
+
+const (
+	_docstringOpen  = "/**"
+	_docstringClose = "*/"
+)
+
+// CleanupDocstring takes a docstring in the form,
+//
+//  /**
+//   * foo bar
+//   */
+//
+// And returns,
+//
+//  foo bar
+func CleanupDocstring(s string) string {
```

Comment:
Cleanup feels weird here, suggest `ParseDocstring` instead.

Reply:
Or maybe `FormatDocstring`?

Reply:
I like Parse more than Format here. This isn't formatting a docstring to be printed but parsing what the actual text from a docstring.